### PR TITLE
(MS #14) Create a new automated, cross-platform test framework

### PR DESCRIPTION
**Pull request by HowardRoeske**
https://github.com/PlayFab/XPlatBetaSdk/pull/31

### CODE REVIEW

File: targets/XPlatCppSdk/source/test/TestApp/PlayFabApiTest.h [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+#pragma once
+
+#include <functional>
+#include <string>
+#include "TestCase.h"
+
+namespace PlayFab
+{
+    struct PlayFabError;
+    class PlayFabEventAPI;
```
Comment (Vicodex) [inline targets/XPlatCppSdk/source/test/TestApp/PlayFabApiTest.h]: >PlayFabEventAPI [](start = 10, length = 15)

This possibly is not needed in this file

Comment (pgilmorepf) [reply targets/XPlatCppSdk/source/test/TestApp/PlayFabApiTest.h]: Confirmed, doesn't seem necessary.

**Comment by pgilmorepf** [comment]
It would be nice if you removed the one line, PlayFabEventAPI forward decl, that Victor found, but no review is required after that.

**Comment by HowardRoeske** [comment]
Forward declaration has been removed.

**Pull request merged**